conf/application.conf

```diff
@@ -190,6 +190,7 @@ features {
   xi-eori-enabaled: true
   cash-account-v2-enabled: false
   home-page-links-enabled: false
+  authorities-notification-panel-enabled: false
```

Comment:
It didn;t mention in the ticket that we are hiding the notification with a feature flag - I thought it would just be removal of the code?

Reply:
Could you update the ticket to reflect that please?